Library/AccountAcquisition.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+    public class AccountAcquisition : RecurlyEntity
+    {
+        public enum AccountAcquisitionChannel : short
+        {
+            Referral,
+            SocialMedia,
+            Email,
+            PaidSearch,
+            OrganicSearch,
+            DirectTraffic,
+            MarketingContent,
+            Blog,
+            Events,
+            OutboundSales,
+            Advertising,
+            PublicRelations,
+            Other
+        }
+
+        public string AccountCode { get; private set; }
+        public int? CostInCents { get; set; }
+        public string Currency { get; set; }
+        public AccountAcquisitionChannel? Channel { get; set; }
+        public string SubChannel { get; set; }
+        public string Campaign { get; set; }
+
+        private const string UrlPrefix = "/accounts/";
+        private const string UrlPostfix = "/acquisition";
+
+        public AccountAcquisition(string accountCode) : this()
```

Comment:
Perhaps we want to add Currency to these since it's mandatory?

Reply:
According to the docs, Currency is optional. https://dev.recurly.com/docs/create-account-acquisition
While testing I've run into the issue if Currency not set. Currently, I can't reproduce it.
I've modified CreateAccountAcquisitionWithMandatoryParameters test.